bindings/generator/api/client.py

```diff
@@ -149,3 +153,21 @@ async def client_workspace_share(
     role: Optional[RealmRole],
 ) -> Result[None, ClientWorkspaceShareError]:
     raise NotImplementedError
+
+
+class ClientWorkspaceInfoError(ErrorVariant):
+    class Internal:
+        pass
+
+
+async def client_info(
+    client: Handle,
+) -> Result[
+    Tuple[OrganizationID, DeviceID, UserID, UserProfile, Optional[HumanHandle]],
```

Comment:
Can we create a struct for client info ?

Reply:
Related https://github.com/Scille/parsec-cloud/pull/5198#discussion_r1324281187

Reply:
Done